contrib/notebooks/deep_learning/09_Clean_Up.ipynb

```diff
@@ -0,0 +1,253 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# IX - Clean Up\n",
+    "In this notebook we will deallocate the Batch pool and remove the Azure resources we created earlier.\n",
+    "\n",
+    "* [Setup](#section1)\n",
+    "* [Deallocate pool](#section2)\n",
+    "* [Delete Azure resources](#section3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='section1'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a simple alias for Batch Shipyard"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%alias shipyard SHIPYARD_CONFIGDIR=config python /home/nbuser/batch-shipyard/shipyard.py $*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check that everything is working"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
```

Comment:
Looks like there's saved output - do you mind going through all of the notebooks and clearing all output cells?